src/tokenizer.rs

```diff
@@ -1004,6 +1004,18 @@ impl<'a> Tokenizer<'a> {
                             }
                         }
                         Some(' ') => Ok(Some(Token::AtSign)),
+                        // Snowflake stage identifier, this should be consumed as multiple dot separated word tokens
+                        Some(_) if dialect_of!(self is SnowflakeDialect) => {
+                            let mut s = "@".to_string();
+                            s.push_str(&peeking_take_while(chars, |ch| {
+                                self.dialect.is_identifier_part(ch)
+                                    || ch == '/'
+                                    || ch == '~'
+                                    || ch == '%'
+                                    || ch == '.'
```

Comment:
I think this line is causing a problem and requires tests to be changed. 

The way I understand a problem is `/` can be either DIV token or part of identifier and this depends on a previous token (if a previous token before `.` started with `@`). I am not sure if this should be solved within tokenizer.  

Perhaps we should let it become DIV and then parse differently `indentifier div identifier` and check if the first identifier starts with `@`. Looks like a problem to be solved at the parsing layer.

Reply:
@pawel-big-lebowski are there more places that '/' is part of identifier except of 'namespace/path'?
I'm not familiar with another places - maybe better approach will be to handle div in `parse_copy_into`?

Reply:
@yuval-illumex supporting `namespace/path` in `parse_copy_into` sounds good to me.

Reply:
I redid the parsing so it tokenizes normally and then I collect all `@` `%` `/` as parts of Ident.